LessOOP/Program.cs

```diff
@@ -68,15 +116,153 @@ static void Main(string[] args)
                     Console.WriteLine($"Вы сняли сумму {sum2} и остаток на балансе { acc.BalansAccounts}");
                 }
 
+                // Списание со счета на счет
+                Console.WriteLine($"Введите номер счета для списания:");
+                var intextNum0 = Console.ReadLine();
+                long numdonor = Convert.ToInt64(intextNum0);
+                Accounts accdonor = Arr[0];//счет донор
+                if (FindeAcc(numdonor, Arr, out accdonor) == true)
+                {
+                    Console.WriteLine($"Введите сумму для снятия:");
+                    intext = Console.ReadLine();
+                    if (intext == "") intext = "0";
+                    decimal sumd = Convert.ToDecimal(intext);
+                    Console.WriteLine($"На балансе счета донора: {accdonor.BalansAccounts}");
+
+                    if (acc.FromAccToAcc(accdonor, sumd) == true)
+                    {
+                        Console.WriteLine($"На балансе счета донора (после): {accdonor.BalansAccounts}");
+                        Console.WriteLine($"На балансе текущуго счета(после): {acc.BalansAccounts}");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Недостаточно средств для перевода со счета");
+                    }
+                }
+
+
+
             }
-            else 
+            else
             {
                 Console.WriteLine($"Счет не найден.");
             }
+        }
+        static bool FindeAcc(long num, Accounts[] arr, out Accounts acc)
```

Comment:
```suggestion
        private static bool FindeAcc(long num, Accounts[] arr, out Accounts acc)
```
Модификатор доступа потерялся.